myhpom/static/myhpom/page/_signup.scss

```diff
@@ -9,3 +9,16 @@
     // border: 1px solid var(--danger);
     padding: 1rem;
 }
+
+.form-group__label--invalid {
+	color: #dc3545;
+}
+
+.form-group-tos__div--invalid {
+	border: 1px solid #dc3545;
+	border-radius: 0.25rem;
+}
+
+.form-group-tos__h5--invalid {
+	color: #dc3545;
```

Comment:
Create a variable for this color, since its repeated a few times. I imagine as we get more from Shane we'll want to make our own theme for bootstrap, but for the time being....defining the magic color codes used more than once seems sensible to me...

Reply:
good idea.